### DISCUSSION TITLE
Which replacement for tinymce?

### DISCUSSION THREAD

**Discussion by luigifab** [OpenMage/magento-lts#1996] [Questions & Answers]
By removing tinymce files: ~113,000 deletions (-)

Who know a modern a simple rich editor that can replace it?
(please, without jQuery)

**Comment by tmotyl** [comment]
https://ckeditor.com/ckeditor-5/

**Comment by addison74** [comment]
If this PR is completed, there will be no need to look for a replacement.

https://github.com/OpenMage/magento-lts/pull/3220
